includes/class-core.php

```diff
@@ -42,6 +42,14 @@ public function __construct() {
 		$this->includes();
 
 		add_action( 'wp_enqueue_scripts', [ __CLASS__, 'enqueue_scripts' ] );
+		register_activation_hook( NEWSPACK_ADS_PLUGIN_FILE, [ __CLASS__, 'activation_hook' ] );
+	}
+
+	/**
+	 * Activation Hook
+	 */
+	public function activation_hook() {
```

Comment:
```suggestion
	public static function activation_hook() {
```

Because otherwise: `Non-static method Newspack_Ads\Core::activation_hook() should not be called statically`.